### ISSUE TITLE
Animation direction arrow selector is not keyboard accessible

### ISSUE THREAD

**Issue by bmattb** [GoogleForCreators/web-stories-wp#5528]
## Bug Description

The animation direction arrow selector (for e.g. fly-in effect) is not keyboard-accessible. Only the currently selected direction is reachable by tab, the other directions are not in the tab-order (probably because they’re disabled). Also, I’d expect arrow keys to be a shortcut for setting the animation direction. The widget should probably be styled as a role=“listbox” with the arrows as role=“option” to follow WCAG. Also, the arrows should have sensible labels.

Currently implemented with a radio group behind the scenes, but it’s not working as such using arrow keys (radio groups can normally be navigated using arrow keys, but we have global listeners preventing that - local listeners will work though).

It can remain a radio group as long as it’s properly announced to screen-readers as such and work as expected for such (including arrow key navigation).

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by BrittanyIRL** [comment]
This is fixed with this: https://github.com/google/web-stories-wp/pull/5574 and https://github.com/google/web-stories-wp/pull/5615

**Comment by BrittanyIRL** [comment]
@csossi for testing - the branch is https://github.com/google/web-stories-wp/pull/5615 and also fixes this issue: https://github.com/google/web-stories-wp/issues/5613

**Comment by csossi** [comment]
Verified in QA
